developer_scripts/update_container_tags.sh

```diff
@@ -9,8 +9,8 @@ if [[ "$target" == "main" ]]; then
   search_pattern='container: "ghcr\.io/stjudecloud/([^:]*):branch-[^-]*-(.*)"'
   replacement='container: "ghcr.io/stjudecloud/\1:\2"'
 else
-  search_pattern='container: "ghcr\.io/stjudecloud/([^:]*):(.*)"'
-  replacement='container: "ghcr.io/stjudecloud/\1:branch-'$target'-\2"'
+  search_pattern='container: "ghcr\.io/stjudecloud/([^:]*):(branch-'$target'-)?(.*)"'
```

Comment:
@a-frantz - This isn't quite perfect, but will prevent the current branch from being inserted multiple times. It will break if a different branch tag is present, but I think that's OK for now.

Reply:
Without having testing it locally, this looks fine. Agree that it's ok to break in that case

Reply:
I actually realized that you're already doing the check I want under `main`. So I just copied the regex from above `branch-[^-]*-(.*)`